deluge/core/torrentmanager.py

```diff
@@ -192,8 +198,8 @@ def archive_file(filename):
 
         # Save the state periodically
         self.save_state_timer.start(200, False)
-        self.save_resume_data_timer.start(190, False)
-        self.prev_status_cleanup_loop.start(10)
+        self.save_resume_data_timer.start(190, False)  # The resume data for torrents that need it
+        self.save_all_resume_data_timer.start(900, False)  # Save resume data for all torrents
```

Comment:
These comments should be moved to where the timers are defined. Should actually be in init docstring but there isn't one yet it...